release-calendar/src/server/repository-service.ts

```diff
@@ -28,6 +28,10 @@ export class RepositoryService {
     this.promotionRepository = connection.getRepository(PromotionEntity);
   }
 
+  getReleaseNames(): Promise<Release[]> {
+    return this.releaseRepository.createQueryBuilder('release').getMany();
+  }
```

Comment:
Rename this to `getReleases()` and the existing `getReleases()` to `getPromotions()`. This includes changes to the api route and the api service.

We should probably rename `getCurrentReleases()` to `getCurrentPromotions()` too.